.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -0,0 +1,29 @@
+name: Bug Report
+description: File a bug report
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+        Please describe what you are trying to achieve and what went wrong.
+        Please include the stack trace of the error (error message) you are
+        witnessing.
+        If possible, please specify the version of `Pharmpy` you are running:
+
+        For R users
+        ```r
+        packageVersion("pharmr")
+        print_pharmpy_version()
+        ```
+
+        For Python users
+        ```sh
+        pharmpy --version
```

Comment:
There is `pharmpy.__version__`  in python

Reply:
OK.

Reply:
Done.